manifests/4.7/elasticsearch-operator.v4.7.0.clusterserviceversion.yaml

```diff
@@ -29,7 +29,7 @@ metadata:
     createdAt: 2019-02-20T08:00:00Z
     support: AOS Cluster Logging, Jaeger
     olm.skipRange: ">=4.5.0-0 <4.7.0"
-    alm-examples: |-
+    olm-examples: |-
```

Comment:
Why is this rename here? The official documentation declares them as `alm-examples`:
https://docs.openshift.com/container-platform/4.6/operators/operator_sdk/osdk-generating-csvs.html#osdk-crds-templates_osdk-generating-csvs

Reply:
I am not sure if this is a typo or not. But 'alm-examples' gives me the following error of running `make generate-bundle`

```
kdir -p bundle; \
/root/go/src/github.com/openshift/elasticsearch-operator/bin/opm-v1.13.6 alpha bundle generate --directory manifests/"4.7" --package elasticsearch-operator --channels "4.7" --default "4.7" --output-dir bundle/; \
find bundle/manifests/ -type f ! \( -name "elasticsearch-operator*.yaml" -o -name "*crd.yaml" \) -delete && \
/root/go/src/github.com/openshift/elasticsearch-operator/bin/operator-sdk-v0.18.2 bundle validate --verbose bundle
INFO[0000] Building annotations.yaml                    
INFO[0000] Generating output manifests directory        
INFO[0000] Writing annotations.yaml in /root/go/src/github.com/openshift/elasticsearch-operator/bundle/metadata 
INFO[0000] Building Dockerfile                          
INFO[0000] Writing bundle.Dockerfile in /root/go/src/github.com/openshift/elasticsearch-operator 
DEBU[0000] Debug logging is set                         
INFO[0000] Found annotations file                        bundle-dir=bundle container-tool=docker
INFO[0000] Could not find optional dependencies file     bundle-dir=bundle container-tool=docker
ERRO[0000] Error: Value error converting YAML to JSON: yaml: line 9: did not find expected ',' or '}': error decoding example CustomResource 
```

Changing to `olm-example` fixes the error.

Reply:
The error is because of introducing:
```
"image": "quay.io/openshift/origin-logging-elasticsearch6:latest",
```
and this has a comma inside.

I assume that you don't see the error with `olm-example` because the marshaller doesn't recognise this field and drops it.

Reply:
Never mind, the error should be related to some misaligned JSON format error. I will fix and change it back to "alm". Thanks.